src/views/token-swap/components/quote-selector/quote-selector.styles.js

```diff
@@ -22,9 +22,6 @@ const useQuoteSelectorStyles = createUseStyles((theme) => ({
   toTokensHeader: {
     flex: "1 1 40%",
   },
-  rewardHeader: {
```

Comment:
this is from token swap, so it should stay. It's' a different kind of reward

Reply:
Thx! Reverted!